src/ui_fsmenu/netsetup_lan.cc

```diff
@@ -278,7 +278,8 @@ void FullscreenMenuNetSetupLAN::change_playername() {
 	if (!InternetGaming::ref().valid_username(playername_.text())) {
 		playername_.set_warning(true);
 		playername_.set_tooltip(_("Enter a valid nickname. This value may contain only "
-		                          "English letters, numbers, and @ . + - _ characters."));
+		                          "English letters, numbers, and @ . + - _ characters "
```

Comment:
Didn't we just disallow `@`?

Reply:
`@` in the middle of the name is still allowed, only at the beginning of the name its forbidden now. We can also forbid it completely if you prefer it that way.
I didn't updated the tooltips on purpose since the sentence is long enough even without writing about this corner case.

Reply:
Let's leave it for now.